packages/cli/commands/project/installDeps.js

```diff
@@ -0,0 +1,78 @@
+const {
+  installPackages,
+  getProjectPackageJsonLocations,
+} = require('../../lib/dependencyManagement');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { getProjectConfig } = require('../../lib/projects');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const path = require('path');
+const { i18n } = require('../../lib/lang');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { getAccountId } = require('../../lib/commonOpts');
+
+const i18nKey = `commands.project.subcommands.installDeps`;
+
+exports.command = 'install-deps [packages..]';
```

Comment:
Are you missing a third period here after packages?

Reply:
I had the same thought!  But `yargs` uses `..` instead of `...` like the JS rest operator 


Docs: https://github.com/yargs/yargs/blob/main/docs/advanced.md#variadic-positional-arguments